{"frames":[{"func":"saveRDS(dds, dds_file)","file":"","line_number":18,"end_line_number":18,"character_number":5,"end_character_number":26},{"func":"gzfile(file, mode)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in gzfile(file, mode) : cannot open the connection\n"}